hydra-node/test/Hydra/Model.hs

```diff
@@ -623,6 +623,11 @@ partyKeys =
     cks <- nub . fmap CardanoSigningKey <$> vectorOf len genSigningKey
     pure $ zip hks cks
 
+genContestationPeriod :: Gen ContestationPeriod
+genContestationPeriod = do
+  i :: Word <- arbitrary `suchThat` (> 0)
```

Comment:
i think i will also limit the upper bound.

Reply:
What value do you think it would make sense for upper bound?

Reply:
idk, like an year? :P